### ISSUE TITLE
Validation order

### ISSUE THREAD

**Issue by lukejagodzinski** [typestack/class-validator#399]
Can I define validation order somehow? I haven't found anything in docs and issues

**Comment by gremo** [comment]
It seems reversed. And I think it should stop at the first error:

```
"constraints": {
"length": "abiCode must be longer than or equal to 5 characters",
"isNotEmpty": "abiCode should not be empty"
}
```

For me, it doesn't make any senso to return "length" constraint violation. At least, it should be configurable (`stopAtFirstError`)

**Comment by lukejagodzinski** [comment]
@gremo where do you use the `constraints` option? In the validator decorator?

**Comment by gremo** [comment]
@lukejagodzinski nope, that is the output of `validate `method. It show that the validation order is reversed.

**Comment by peresbruno** [comment]
Same question here.

**Comment by forsigner** [comment]
Same question here +1

**Comment by Demonell** [comment]
That would be nice option, indeed

**Comment by valdineifer** [comment]
> @gremo I didn't ask about in what order validation was performed but about how to define custom order

Wow, this sounds rude.
The @gremo was right. I change the order of the validation decorators and it worked.
